spec/binaries/chrome_xml_spec.ts

```diff
@@ -65,4 +65,17 @@ describe('chrome xml reader', () => {
       done();
     });
   });
+
+  // This test case covers a bug when all the following conditions were true.
+  //  arch was 64 with multiple major versions available.
+  fit('should note get the 85.0.4183.83, 32-bit version (arch = x64)', (done) => {
```

Comment:
```suggestion
  it('should not get the 85.0.4183.38, 32-bit version (arch = x64)', (done) => {
```

Removing the focused `fit`, fixing a typo.

Reply:
Thank you @lotz!